app/templates/todos/new.plush.html

```diff
@@ -0,0 +1,45 @@
+<div>
+    <div class="todo-content">
+        <div class="mb-3">
+            <nav class="navbar navbar-expand-lg py-3">
+                <a class="navbar-brand mr-4" href="#"><h2>ToDo List</h2></a>
+                <div class="border-left">
+                    <p class="text-white ml-4 mt-3">4 Incompleted Tasks<br><small>Thursday 06, June 2019</small>
+                    </p>
+                </div>
+                <div class="collapse navbar-collapse d-flex" id="navbarNav">
+                    <ul class="navbar-nav ml-auto">
+                        <li class="nav-item">
+                            <a class="nav-link active" href="#">Incomplete Tasks</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Completed Tasks</a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>     
```

Comment:
Let's use a partial since this will be used over and over -- example: /index.plush.html